lms/static/js/learner_dashboard/program_list_factory.js

```diff
@@ -0,0 +1,21 @@
+;(function (define) {
+    'use strict';
+
+    define([
+        'js/learner_dashboard/views/program_list_view',
+        'js/learner_dashboard/views/sidebar_view'
+    ],
+    function (ProgramListView, SidebarView) {
+        return function (options) {
+            var listView = new ProgramListView({
+                el: '.program-cards-container',
+                context: options
+            });
+            
+            var rightPanelView = new SidebarView({
```

Comment:
Duplicate var declarations. There should only be 1, comma separate, var declaration at the top of a function.